common/supplemental/likelySubtags.xml

```diff
@@ -1926,8 +1934,8 @@ not be patched by hand, as any changes made in that fashion may be lost.
 		<!--{ Orokolo; ?; ? } => { Orokolo; Latin; Unknown Region }-->
 		<likelySubtag from="oru" to="oru_Arab_ZZ"/>
 		<!--{ Ormuri; ?; ? } => { Ormuri; Arabic; Unknown Region }-->
-		<likelySubtag from="os" to="os_Cyrl_GE"/>
-		<!--{ Ossetic; ?; ? } => { Ossetic; Cyrillic; Georgia }-->
+		<likelySubtag from="os" to="os_Cyrl_RU"/>
```

Comment:
Geopolitical issue, see below.

Reply:
OK , would we ideally leave the likelySubtag  and default content as is?